Tests/Editor/CoreApi/GameLiftComputeManagerTests.cs

```diff
@@ -37,28 +40,24 @@ public void Setup()
             _awsCredentialsTestProvider = new AwsCredentialsTestProvider();
         }
 
-        private GameLiftComputeManager ArrangeAnywhereFleetHappyPath()
+        private GameLiftComputeManager ArrangeRegisterComputeHappyPath()
         {
             var listLocationModel = new List<LocationModel>();
             listLocationModel.Add(new LocationModel
             {
                 LocationName = "custom-location-1"
             });
 
-            _coreApiMock.Setup(f => f.PutSetting(It.IsAny<SettingsKeys>(), It.IsAny<string>())).Returns(Response.Ok(new PutSettingResponse()));
```

Comment:
Do we still need the core api mock? I see the field still exists

Reply:
There was 1 use for it on line 68, but the tests seem to run without the setup so we will not need it. Removed